src/schema/rules/sidecars/mri.yaml

```diff
@@ -35,6 +35,14 @@ MRIHardware:
     MatrixCoilMode: recommended
     CoilCombinationMethod: recommended
 
+MRISample:
+  selectors:
+    - modality == "mri"
+  fields:
+    BodyPart: optional
```

Comment:
Why make BodyPart optional for MRI, but recommended for PET?

Reply:
Unclear. @Remi-Gau?

Reply:
can't remember
would say let's make them optional for both

Reply:
Should the DICOM tag be mentioned in MRI as well?